jovian/jovian_nb_ext/main.js

```diff
@@ -29,49 +29,50 @@ define([
           resolve(data.content.text.trim());
         };
 
-        const nb_filename = Jupyter.notebook.notebook_name;
-        var commit = '\tcommit(nb_filename="' + nb_filename + '")\n';
-
+        const nb_filename = Jupyter.notebook.notebook_name.substr(
+          0,
+          Jupyter.notebook.notebook_name.length - 6
```

Comment:
You can display with the the extension. If you need remove the file extension you can use something like split instead.